### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by lynn915**

### CODE REVIEW

File: operations/workplace/people/performance-reviews-50/README.md [hunk 1]
```diff
@@ -45,33 +45,31 @@ Note: HR will try and limit peer review panel requests to a maximum of 5 per per
...
 
-Team members joining in August will not need to do the [90 day New Colleague review](https://handbook.mattermost.com/contributors/onboarding#new-colleague-90-day-feedback-process) and will instead take part in review cycle. Team members joining September - December will follow the typical [90 day New Colleague review process](https://handbook.mattermost.com/contributors/onboarding#new-colleague-90-day-feedback-process) and are not required to complete the End of Year Review.
+Team members joining in August and September will not need to do the [90 day New Colleague review](https://handbook.mattermost.com/contributors/onboarding#new-colleague-90-day-feedback-process) and will instead take part in review cycle. Team members joining October - December will follow the typical [90 day New Colleague review process](https://handbook.mattermost.com/contributors/onboarding#new-colleague-90-day-feedback-process) and are not required to complete the End of Year Review.
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/README.md:61]:
```suggestion
Team members joining in August and September will not need to complete the [90 day New Colleague review](https://handbook.mattermost.com/contributors/onboarding#new-colleague-90-day-feedback-process) and will instead take part in the next review cycle. Team members joining in October through December will follow the typical [90 day New Colleague review process](https://handbook.mattermost.com/contributors/onboarding#new-colleague-90-day-feedback-process) and are not required to complete the End of Year Review.
```

File: operations/workplace/people/performance-reviews-50/README.md [hunk 2]
```diff
@@ -45,33 +45,31 @@ Note: HR will try and limit peer review panel requests to a maximum of 5 per per
...
 
 ### Rating Scale
 
 Behaviorally-anchored performance ratings help us to establish a shared and agreed-upon language that describes well-defined performance expectations, desired behaviours, and outcomes. Rating scales can also check biases and enhance distributive and procedural fairness during performance calibrations.
 
-Mattermost previously used a 3-point rating scale. Based on feedback from participants in prior review cycles, this is being expanded for the 2021 Mid-Year Reviews to a 5-point rating scale, increasing granularity around "exceeds expectations" and "meets expectations".
+We have had several iterations of the rating scale as we continue to respond to staff feedback. For the EOY 2021 review cycle, we moved to a 4-point rating scale to ensure clarity and differentiation between ratings. Based on feedback, we have also adjusted the rating scale to have 1 clear rating of "not meeting expectations" and 3 options describing average to above-average performance.
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/README.md:67]:
```suggestion
We have iterated through several iterations of the rating scale as we continue to respond to staff feedback. For the EOY 2021 review cycle, we moved to a 4-point rating scale to ensure clarity and differentiation between ratings. Based on feedback, we have also adjusted the rating scale to have one clear rating of "not meeting expectations" and three options describing average to above-average performance. 
```

File: operations/workplace/people/performance-reviews-50/README.md [hunk 3]
```diff
@@ -83,4 +81,4 @@ Yes. In the process of reviewing performance against target levels, changes in l
 
 * Learn how to use Culture Amp, nominate peer reviewers, give effective peer and upward feedback, and write your self-reflection in Culture Amp's [self-guided course](https://www.cultureamptraining.com/performance-reviews-for-employees).
 * People Managers should take Culture Amp's [self-guided course](https://www.cultureamptraining.com/performance-for-managers) to learn how to use the platform as a people manager. The course includes tips for writing evaluations, giving clear and actionable feedback, and helping your direct reports write strong goals.
-
+* See[EOY Review Toolkit](https://docs.google.com/document/d/1Jj62uswZ-3yPb0gd-ixfzVKCMwv7vO2AVDLdG8N1eyw/edit?usp=sharing) for additional resources.
```
Comment (cwarnermm) [inline operations/workplace/people/performance-reviews-50/README.md:84]:
```suggestion
* See the [EOY Review Toolkit](https://docs.google.com/document/d/1Jj62uswZ-3yPb0gd-ixfzVKCMwv7vO2AVDLdG8N1eyw/edit?usp=sharing) for additional resources.
```

**Review by cwarnermm — COMMENTED** [review]
Great update! Feedback provided inline.

**Review by lynn915 — COMMENTED** [review]
This looks great, @cwarnermm . You can publish this. Thanks!

**Review by cwarnermm — APPROVED** [review]

**Pull request merged**